products/opensuse/main.pm

```diff
@@ -236,6 +236,9 @@ sub load_inst_tests {
         if (defined(get_var("RAIDLEVEL"))) {
             loadtest "installation/partitioning_raid";
         }
+        elsif (check_var('LVM', 0) && get_var('ENCRYPT')) {
```

Comment:
corresponds to `!get_var('LVM')`, isn't it? And that would be a more common style

Reply:
!get_var will be true when variable is not set, as I didn't want to introduce new variable I suggest setting LVM to 0 explicitly. Otherwise we should introduce new variable like NO_LVM, Problem is that ENCRYPT flag is used in combinations with other settings as well, not only LVM, so it can be the case that LVM is not set, but ENCRYPT is and we don't want this scenario to be executed.